packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -763,8 +763,7 @@ const styles = StyleSheet.create({
      */
     button: {
         position: "absolute",
-        right: sizing.size_040,
-        top: sizing.size_040,
+        insetInlineEnd: sizing.size_040,
```

Comment:
Should this have `insetInlineBlock` as well? I'm seeing a shift in Chromatic that might be caused by this.